tests/sentry/api/endpoints/test_organization_code_mappings.py

```diff
@@ -232,6 +232,11 @@ def test_basic_post_with_valid_integrationId(self):
             "defaultBranch": "master",
         }
 
+    def test_basic_post_from_member_permissions(self):
+        self.login_as(user=self.user2)
+        response = self.make_post()
+        assert response.status_code == 201, response.content
```

Comment:
This creates a new code mapping by a non-superuser user:
https://github.com/getsentry/sentry/blob/c605f45a3ca0e585cd7ca1c5a0c286c0d61913e3/tests/sentry/api/endpoints/test_organization_code_mappings.py#L15